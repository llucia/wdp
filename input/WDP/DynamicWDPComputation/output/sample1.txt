8	63.0
10	64.0
9	66.0
11	56.0
13	90.0
12	67.0
14	91.0

0	50.0
2	84.0
1	60.0
3	76.0
6	50.0
5	30.0
7	40.0
4	111.0
