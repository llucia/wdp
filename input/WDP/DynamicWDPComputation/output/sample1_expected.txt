1 10
2 20
3 30
4 40
5 50
301 40
302 50
401 50
402 60
501 80
502 90
5001 90
5002 100
5003 100
5004 110
